docs/user/dashboard/advanced-editors.asciidoc

```diff
@@ -0,0 +1,65 @@
+[[add-panels-with-advanced-editors]]
+== Create panels with the advanced editors
+
+{kib} provides you with three advanced editors that you can use to manually create dashboard panels.
+
+[float]
+[[tsvb-advanced-editor]]
+=== TSVB
+
+*TSVB* is a time series data visualization editor that allows you to use the full power of the {es} aggregation framework. To use *TSVB*,
```

Comment:
Is "To use TSVB" correct here?  Or does the rest of the sentence show you want TSVB can do.